package-lock.json

```diff
@@ -1,36 +1,36 @@
 {
   "name": "business-create-ui",
-  "version": "5.10.11",
+  "version": "5.10.12",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "business-create-ui",
-      "version": "5.10.11",
+      "version": "5.10.12",
       "dependencies": {
         "@babel/compat-data": "^7.21.5",
         "@bcrs-shared-components/approval-type": "1.0.19",
         "@bcrs-shared-components/base-address": "2.0.3",
         "@bcrs-shared-components/breadcrumb": "2.1.15",
-        "@bcrs-shared-components/business-lookup": "1.3.1",
+        "@bcrs-shared-components/business-lookup": "1.3.3",
         "@bcrs-shared-components/certify": "2.1.15",
         "@bcrs-shared-components/completing-party": "2.1.30",
         "@bcrs-shared-components/confirm-dialog": "1.2.1",
         "@bcrs-shared-components/contact-info": "1.2.15",
         "@bcrs-shared-components/corp-type-module": "1.0.16",
         "@bcrs-shared-components/correct-name": "1.0.46",
         "@bcrs-shared-components/court-order-poa": "3.0.11",
-        "@bcrs-shared-components/date-picker": "1.2.15",
+        "@bcrs-shared-components/date-picker": "1.2.55",
```

Comment:
I've updated this again so that DatePicker honour the input rules (which were previously conditional). I tested this in other filings and it looks fine.